homework/style.css

```diff
@@ -1,3 +1,108 @@
 .alert-error {
```

Comment:
While your app is responsive, the select element needs some margin on the right:

![image](https://user-images.githubusercontent.com/2788771/59962713-540deb80-94e9-11e9-8c00-34f10124a08e.png)

Reply:
I could figure out with 'min-width'